src/main/java/com/ombremoon/enderring/common/init/entity/StatusEffectInit.java

```diff
@@ -71,39 +72,40 @@ public class StatusEffectInit {
                                     "GREENSPILL_CRYSTAL",
                                     10F, AttributeModifier.Operation.ADDITION),
                             EpicFightAttributes.MAX_STAMINA
-                    ).build());
+                    )
+                    .addTier("f8b61f3c-04cc-64b3-6b08-67b8bdb40834",1,0.15f).build());
```

Comment:
This seems like improper usage. I don't know what the previous tier was since the effect says greenspill and that should multiply +15% max stamina, but the previous tier is an addition operation.